target/arc/arconnect.c

```diff
@@ -355,6 +355,8 @@ arconnect_regs_get(const struct arc_aux_reg_detail *aux_reg_detail, void *data)
     case AUX_ID_mcip_bcr:
         return 0x00800000 /* IDU */
                | 0x00040000;
+    case AUX_ID_mcip_idu_bcr:
+	return 0x300;
```

Comment:
I don't like magic values, let's try to at least document what is encoded there, even if in-place describe the structure of that value.